packages/@aws-containers/aws-ecs-builder/lib/environment.ts

```diff
@@ -0,0 +1,82 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as ecs from '@aws-cdk/aws-ecs';
+import * as cdk from '@aws-cdk/core';
+import { EnvironmentCapacityType } from './extensions/extension-interfaces';
+
+/**
+ * Settings for the deploy environment you want to deploy
+ * services within.
+ */
+export interface EnvironmentProps {
+  /**
+   * The VPC used by the service for networking
+   * @default - Create a new VPC
+   */
+  readonly vpc?: ec2.IVpc,
```

Comment:
Have you given any thought to what it might look like connecting two Environments together? Seems like another area where we could expose a few new patterns, such as a VPC endpoint and App Mesh gateway.